modules/sso.php

```diff
@@ -1002,8 +1020,16 @@ public function error_msg_email_already_exists( $message ) {
 	 * @return string
 	 **/
 	public function msg_login_by_jetpack( $message ) {
-
-		$msg = __( sprintf( 'Jetpack authenticates through WordPress.com — to log in, enter your WordPress.com username and password, or <a href="%1$s" target="_blank">visit WordPress.com</a> to create a free account now.', 'http://wordpress.com/signup' ) , 'jetpack' );
+		$msg = sprintf(
+			wp_kses(
+				__(
```

Comment:
I think this message should cater more to needing a WordPress.com account rather than mentioning 'Jetpack authenticates...'

How about 

"A WordPress.com account is required to access this site. Click the button below to sign in or create a free WordPress.co account."